examples/run_script_with_local_browser/README.md

```diff
@@ -0,0 +1,37 @@
+# Example script: run the script in existing local browser with AgentQL
+
+This example demonstrates how to run the script in an existing local browser with AgentQL.
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/docs/installation/sdk-installation)
+- Save this python file locally as **run_script_with_local_browser.py**
+- Close your Google Chrome browser application if it is open.
+- If you're using **Mac**, open the terminal and run the following command:
+
+```bash
+/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=9222
+```
+
+- If you're using **Windows**, open the Command Prompt and run the command:
+
+```bash
+chrome.exe --remote-debugging-port=9222
+```
+
+**Make sure to replace chrome.exe with the path to your Chrome executable if it's not already in your system's PATH.**
+
+- In the open browser, select the Google profile you would like to use for this session.
```

Comment:
Not sure what did you mean, but suggestions might be: "open browser dialog" or "In opened browser"?